### DISCUSSION TITLE
windows版的一些改进需求

### DISCUSSION THREAD

**Discussion by auh315** [xiaoyaocz/flutter_dmzj#108] [General]
1. 现在版本不支持鼠标滚轮翻页，好像只能点击页面边缘翻页和键盘上下键？这点我不是很确定。从右往左往后翻页的时候还是只能用↑键，感觉有点反直觉。然后从右往左翻页的话进度条最好也得反个方向
2. 不支持双页阅读……pc端这个真的挺重要的，感觉可以参考一下tachiyomi2k和comic read脚本
3. 全屏好像也不是很完善，打开设置页面的全屏以后，没找到隐藏客户端边缘的方法，常见的F、F11、ENTER键好像都没全屏
4. 一点私心，希望能加一个关掉吐槽的设置，就是设置下面那个吐槽按钮还带红色数字，emmm，总想点进去看看😂

以上都是用惯本地漫画阅读器的一些吐槽，嘛，不知道在线客户端能不能改就是了

**Comment by xiaoyaocz** [comment]
桌面平台还需要很多改进，现在很多还是触屏的操作逻辑

**Reply by fkiliver** [reply]
> 桌面平台还需要很多改进，现在很多还是触屏的操作逻辑

最好是给选择，win平板需要触屏逻辑
